next/docs/iota-sdk/docs/how-tos/exchange-guide.mdx

```diff
@@ -0,0 +1,243 @@
+
+:::note
+
+If you are not familiar with the IOTA SDK, you can find more information in the [documentation](../welcome.md).
+
+:::
+
+You can use the following examples as a guide to implementing _the multi-account approach_ using the `NodeJs` bindings:
+
+1. [Set up the the IOTA SDK](#1-set-up-the-iota-sdk).
+2. [Create an account for each user](#2-create-an-account-for-each-user).
+3. [Generate a user address to deposit funds](#3-generate-a-user-address-to-deposit-funds).
+4. [Check the user balance](#4-check-the-account-balance).
+5. [Listen to events](#5-listen-to-events).
+6. [Enable withdrawals](#6-enable-withdrawals).
+
+:::note
+
+If you are looking for other languages, please read the [IOTA SDK overview](../welcome/#your-application-in-the-iota-network).
+
+:::
+
+Since all IOTA SDK bindings are based on core principles provided by the IOTA SDK, the outlined approach is very similar regardless of the programming language you choose.
+
+### 1. Set Up the IOTA SDK
+
+First, you should install the components that are needed to use the IOTA SDK and the binding of your choice; it may vary a bit from language to language. In the case of the [NodeJs binding](../getting-started/nodejs.mdx), it is straightforward since it is distributed via the `npm` package manager.
+
+You can read more about [backup and security in this guide](https://wiki.iota.org/shimmer/introduction/how_tos/backup_security).
+
+```bash
+npm install @iota/sdk dotenv
+```
+
+:::warning
+
+We use `dotenv` to share user-defined constants across many examples. This is for convenience only and shouldn't be done in production.
+
+:::
+
+You can create a `.env` by running the following command:
+
+```bash
+touch .env
+```
+
+The following examples will need these variables to exist, here are some default values you can use but also change:
+
+```bash
+WALLET_DB_PATH="./wallet-database"
+NODE_URL="https://api.testnet.shimmer.network"
+STRONGHOLD_SNAPSHOT_PATH="./wallet.stronghold"
+STRONGHOLD_PASSWORD="*K6%79#yeFn7AMQhMQ"
+EXPLORER_URL="https://explorer.shimmer.network/testnet"
+```
+
+#### 1 Generate a mnemonic
+
+```js reference
+https://github.com/iotaledger/iota-sdk/tree/develop/bindings/nodejs/examples/exchange/0-generate-mnemonic.ts
+```
+
+After you have updated the `.env` file, you can initialize the `Wallet` instance with a secret manager([`Stronghold`](https://wiki.iota.org/stronghold.rs/welcome) by default) and client options.
+
+:::note
+
+Manage your password with the utmost care.
+
+:::
+
+By default, the Stronghold file will be called `wallet.stronghold`. It will store the seed (derived from the mnemonic) that serves as a cryptographic key from which all accounts and related addresses are generated.
```

Comment:
```suggestion
By default, the Stronghold file will be called `example.stronghold`. It will store the seed (derived from the mnemonic) that serves as a cryptographic key from which all accounts and related addresses are generated.
```